processor/processor.go

```diff
@@ -318,6 +318,8 @@ func (p *processor) processStructFields(parentType *types.Type, pkg *loader.Pack
 			Name:     f.Name,
 			Doc:      f.Doc,
 			Embedded: f.Name == "",
+			Markers:  f.Markers,
```

Comment:
This doesn't seem to be working properly. When I go to access markers in my own templates, the markers don't seem to be there. I'm still familiarizing myself w/ https://pkg.go.dev/sigs.k8s.io/controller-tools/pkg/markers, but maybe this has something to do w/ how the Collector or Registry are setup?

Reply:
It seems like markers have to defined before parsing. That's obviously problematic given I wanted to wire all markers without knowing what they are ahead of time